views/scan.ejs

```diff
@@ -252,13 +300,13 @@
 
         if (data.pageNum > 1) {
           $('.page-link-previous')
-            .attr('href', '?startKey=' + data.prevKey + '&pageNum=' + (data.pageNum-1) + '&'  + data.filterQueryString)
-          $('.page-item-previous').removeClass('disabled')
+            .attr('href', '?startKey=' + data.prevKey + '&pageNum=' + (data.pageNum-1) + '&' + data.filterQueryString)
+         $('.page-item-previous').removeClass('disabled')
         }
 
         if (data.nextKey) {
           $('.page-link-next')
-            .attr('href', '?startKey=' + data.nextKey + '&prevKey=' + data.prevKey + '&pageNum=' + (data.pageNum+1) + '&'  + data.filterQueryString)
+            .attr('href', '?startKey=' + data.nextKey + '&prevKey=' + data.prevKey + '&pageNum=' + (data.pageNum+1) + '&' + data.filterQueryString)
           $('.page-item-next').removeClass('disabled')
         }
```

Comment:
A bit below please replace `showing items` with just `showing` as now `items` is redundant with `Items` in front of it already.